featuretools/synthesis/deep_feature_synthesis.py

```diff
@@ -491,16 +498,15 @@ def _build_transform_features(self, all_features, entity, max_depth=0):
                                          new_feature=new_f)
 
     def _build_forward_features(self, all_features, parent_entity,
-                                child_entity, relationship, max_depth=0):
+                                child_entity, relationship, max_depth=0,
+                                allowed_variable_types=None):
 
         if max_depth is not None and max_depth < 0:
             return
 
         features = self._features_by_type(all_features=all_features,
                                           entity=parent_entity,
-                                          variable_type=[Numeric,
-                                                         Categorical,
-                                                         Ordinal],
+                                          variable_type=allowed_variable_types,
```

Comment:
I think we actually shouldn't be filtering any features based on type here since happens on line 216.

That means the correct change here would be to actually to update  `_features_by_type` to not filter based on variable_type if `variable_type` == None.

Reply:
Good point. I'll update this.